concent_api/core/decorators.py

```diff
@@ -122,8 +124,9 @@ def wrapper(request: HttpRequest, *args: Any, **kwargs: Any) -> Union[HttpRespon
                 log(logger, 'Error:', exception.__class__.__name__)
                 return JsonResponse({'error': f'{exception}'}, status = 400)
             except MessageError as exception:
-                log(logger, ERROR_IN_GOLEM_MESSAGE, exception.__class__.__name__)
-                return JsonResponse({'error': join_messages(ERROR_IN_GOLEM_MESSAGE, str(exception))}, status = 400)
+                request_id = generate_uuid()
+                log(logger, f'uuid: {request_id}', ERROR_IN_GOLEM_MESSAGE, exception.__class__.__name__, str(traceback.format_exc()))
```

Comment:
```suggestion
                log(logger, f'uuid: {request_id}', ERROR_IN_GOLEM_MESSAGE, str(exception), log_level=exception
```

this would require adding new log level to `LoggingLevel` and adapt `log` function